scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -457,13 +457,26 @@ class ScioContext private[scio] (val options: PipelineOptions,
    * Get an SCollection for a Datastore query.
    * @group input
    */
-  def datastore(datasetId: String, query: Query): SCollection[Entity] = pipelineOp {
-    if (this.isTest) {
-      this.getTestInput(DatastoreIO(datasetId, query))
-    } else {
-      wrap(this.applyInternal(gio.DatastoreIO.readFrom(datasetId, query)))
+  def datastore(datasetId: String, query: Query): SCollection[Entity] =
```

Comment:
I thought (datasetId, namespace, query) looks like a more natural ordering for this. Yet I can make the change if you insist. Can we keep it with this order?